src/routes/+layout.svelte

```diff
@@ -89,9 +89,6 @@
 {:else}
 	<div class="page-container">
 		<Navbar />
-		<div class="redirect-container">
-			<p><a href="/launchpad">Join Papaya SNS 🥭</a></p>
```

Comment:
Was fun while it lasted 🙃

Reply:
@uk4z it's just occurred to me while walking to work, shouldn't this be left in place until all Papaya WTN has been claimed (maybe just renamed in the meantime)? I expect many users to panic about their investment otherwise 😅